FinSearchUnified/Resources/snippets/frontend/search/query_info_message.ini

```diff
@@ -1,16 +1,9 @@
 [en_GB]
-frontend/search/query_info_message/cat = 'Search results for {$finQueryInfoMessage.filter_name} <strong>{$finQueryInfoMessage.cat}</strong> (<strong>{$sSearchResults.sArticlesCount}</strong> hits)'
-frontend/search/query_info_message/vendor = 'Search results for {$finQueryInfoMessage.filter_name} <strong>{$finQueryInfoMessage.vendor}</strong> (<strong>{$sSearchResults.sArticlesCount}</strong> hits)'
-frontend/search/query_info_message/query = 'Search results for {$finQueryInfoMessage.query} (<strong>{$sSearchResults.sArticlesCount}</strong> hits)'
+frontend/search/query_info_message/filter = 'Search results for {$finQueryInfoMessage.filter_name} <strong>"{$finQueryInfoMessage.filter_value}"</strong> (<strong>{$sSearchResults.sArticlesCount}</strong> hits)'
+frontend/search/query_info_message/query = 'Search results for "{$finQueryInfoMessage.query}" (<strong>{$sSearchResults.sArticlesCount}</strong> hits)'
```

Comment:
Please remove all quotes (`"`) in the query info message translations. We wanted to do this in another story, but since this was changed anyway, we can do it here rightaway.